app/models/loader/types.rb

```diff
@@ -170,17 +185,34 @@ def identifier
     class Group < Record
       def_delegators :@policy_object, :gidnumber
 
+      def verify; end
```

Comment:
Shouldn't this also call `super` so that the verify of `Record` will run as well?

Reply:
No - it will through an exception then